pkgs/tools/wayland/proycon-wayout/remove-werror.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/meson.build b/meson.build
+index 2a0ca5f..a4f7a51 100644
+--- a/meson.build
++++ b/meson.build
+@@ -6,7 +6,6 @@ project(
+   default_options: [
+     'c_std=c11',
+     'warning_level=3',
+-    'werror=true',
```

Comment:
This should be done with substiteInPlace in postPatch to avoid patch apply conflicts.

Reply:
I generally prefer patches over find-and-replace for durability reasons, but I can see why search-and-replace would be convenient for maintainers. Happy to comply with repo conventions.

Patch has been replaced with substituteInPlace.